docs/chains/darwinia-smart-chain.md

```diff
@@ -0,0 +1,101 @@
+---
+sidebar_position: 2
+title: Darwinia Smart Chain
+---
+
+## Introduction
+
+Darwinia Smart Chain is a substrate based ethereum compatible smart contract platform where contract developers can easily migrate
+smart contracts from the Ethereum chain to run their business. In this article we will cover a number of topics that are important to contract developers, such as network configuration, pre-compiles and so on. In addition, we have set up a free test network, the pangoro network, where the latest features will be fully tested before they are added to the Darwinia Smart Chain. The user can obtain test tokens at [Pangoro Test Token Faucet](https://apps.darwinia.network/?network=pangoro).
+
+###  Network RPC Configuration
+
+#### For Darwinia Smart Chain
+
+- Network Name: `Darwinia`
+- Token Name: `RING`
+- Token Decimal: `18`
+- ChainId: `46`
+- Block Explorer URL: `https://darwinia.subscan.io/`
+- HTTP Endpoints:
+    - `https://crab-rpc.darwinia.network`
+    - `https://darwinia-rpc.dwellir.com`
+- WSS Endpoints:
+    - `wss://rpc.darwinia.network`
+    - `wss://darwinia-rpc.dwellir.com`
+
+#### For Pangoro Smart Chain
+
+- Network Name: `Pangoro`
+- Token Name: `ORING`
+- Token Decimal: `18`
+- ChainId: `45`
+- Block Explorer URL: `https://pangoro.subscan.io/`
+- HTTP Endpoints:
+    - `https://pangoro-rpc.darwinia.network`
+- WSS Endpoints:
+    - `wss://pangoro-rpc.darwinia.network`
+
+
+### Pre-compiles
```

Comment:
Precompiled Contracts or Precompile, according to EIP name convention.

Reply:
Make sense. I'll update this later.